modules/tinymce/src/themes/silver/main/ts/ui/menus/item/structure/ItemStructure.ts

```diff
@@ -139,7 +139,8 @@ const rtlIcon = [
   'list-num-lower-greek',
   'list-num-lower-roman',
   'list-num-upper-alpha',
-  'list-num-upper-roman'
+  'list-num-upper-roman',
+  'table-row-numbering'
```

Comment:
This really shouldn't be duplicated and probably should be inside `Icons.ts`, but that's outside of scope for this Jira.

Reply:
Unrelated to this change but I also noticed these lists are different. That's definitely a bug as the icons can be used in both menus and toolbars.

Reply:
A Jira has been created to look into this: TINY-7782